python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -44,6 +62,48 @@ def merge_lod_tensor(in_true, in_false, x, mask, level=0):
     return out
 
 
+def Print(input,
+          first_n=-1,
+          message=None,
+          summarize=-1,
+          print_tensor_name=True,
+          print_tensor_type=True,
+          print_tensor_shape=True,
+          print_tensor_lod=True):
+    '''
+    Creates a print op that will print when a tensor is accessed.
```

Comment:
Make sure that the layer comments are same as the convention defined in https://github.com/PaddlePaddle/Paddle/pull/6806

Reply:
done